### DISCUSSION TITLE
How to reduce GPU consumption for loading live2d models in Pixijs

### DISCUSSION THREAD

**Discussion by linhanxiao** [pixijs/pixijs#8285] [Q&A]
Thanks for sharing a very useful project!

I have a project here that uses face motion capture to drive a live2d model, which uses the pixi-live2d-display library and AI model calculation, but later found that most of the cpu, GPU, and memory consumption are For pixi to create application objects and add live2d models to the stage, about 25% of the GPU is used, but the AI ​​face feature calculation consumes only 14% of the GPU.

So the question is, is there some way to significantly reduce GPU resource consumption both in terms of creating application objects in pixi and adding live2d models to the stage?

Looking forward to your reply @GoodBoyDigital

**Comment by miltoncandelero** [comment]
I assume love2d uses a single texture? maybe you can try with a particle container? However I am not sure how the display tree of a live2d model looks like 🤔
